### PR TITLE
Install code-server via conda

### PR DESCRIPTION
**Pull request by xhochy**
Try it: https://mybinder.org/v2/gh/xhochy/vscode-binder/code-server-conda?urlpath=vscode

### CODE REVIEW

**Comment by elgalu** [comment]
works

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+dependencies:
+  - numpy
+  - jupyter-server-proxy
+  - "jupyterlab >0.34.9"
```
Comment (betatim) [inline environment.yml]: Do we need to explicitly mention jupyterlab? Should be installed "as default" by repo2docker and should be v1.x so we are good there as well.

File: postBuild [hunk 2]
```diff
@@ -1,21 +1,11 @@
...
-# Download and install VS Code server
-wget -q -O code-server.tar.gz https://github.com/cdr/code-server/releases/download/${CODE_SERVER_VERSION}/code-server${CODE_SERVER_VERSION}-linux-x86_64.tar.gz
-mkdir -p ~/.local/bin
-tar -xvf code-server.tar.gz -C ~/.local/bin --strip-components=1
-rm -rf code-server.tar.gz
-
-wget https://github.com/microsoft/vscode-python/releases/download/2019.10.44104/ms-python-release.vsix
-code-server --install-extension ms-python-release.vsix
-rm -rf ms-python-release.vsix
+code-server --install-extension ms-python.python
```
Comment (betatim) [inline postBuild:8]: Did the name of the extension change? (why did we go from `ms-python-release.vsix` to `ms-python.python`?)

**Comment by betatim** [comment]
Nice! Two short questions and then good to merge.

File: environment.yml (hunk 1 above)
Comment (xhochy) [reply environment.yml]: I just ported the `requiurements.txt` over here, you probably know better than me why this line is in here.

File: postBuild (hunk 2 above)
Comment (xhochy) [reply postBuild:8]: No, these are different names.

- `vscode-python` is the name of the extension
- `ms-python-release.vsix` is the name of the *release* tarball on Github
- `ms-python.python` is the id in the code-server extension marketplace.

File: environment.yml (hunk 1 above)
Comment (betatim) [reply environment.yml]: Then we should delete it because it is "legacy".

Comment (betatim) [inline environment.yml]:
```suggestion
```

**Comment by betatim** [comment]
One more thing: we can also ditch the `apt.txt` as the conda package will pull in all the dependencies we need to run code-server.

**Comment by betatim** [comment]
Diff stats that are cool: +7-16 :D

Merging!

**Pull request merged**